README.md

```diff
@@ -109,51 +194,67 @@ default ✓ [======================================] 10 VUs  1m0s
+| **SREM**        | `srem(key: string, members: any[]) => Promise<number>`    | Removes the specified members from the set stored at `key`. Specified members that are not a member of this set are ignored. If key does not exist, it is treated as an empty set and this command returns 0. | On **success**, the promise **resolves** with the number of members that were remove from the set, not including non existing members.              |
+| **SISMEMBER**   | `sismember(key: string, member: any) => Promise<boolean>` | Returns if member is a member of the set stored at `key`.                                                                                                                                                     | On **success**, the promise **resolves** with `true` if the element is a member of the set, `false` otherwise.                                      |
+| **SMEMBERS**    | `smembers(key: string) => Promise<string[]>`              | Returns all the members of the set values stored at `keys`.                                                                                                                                                   | On **success**, the promise **resolves** with an array containing the values present in the set.                                                    |
+| **SRANDMEMBER** | `srandmember(key: string) => Promise<string>`             | Returns a random element from the set value stored at `key`.                                                                                                                                                  | On **success**, the promise **resolves** with the selected random member. If the set does not exist, the promise is **rejected** with an error.     |
+| **SPOP**        | `spop(key: string) => Promise<string>`                    | Removes and returns a random element from the set value stored at `key`.                                                                                                                                      | On **success**, the promise **resolves** to the returned set member. If the set does not exist, the promise is **rejected** with an error.          |
 
 ### Custom operations
 
-In the event a redis command you wish to use is not implemented yet, the `sendCommand(command: String, args: Array<any>) -> Promise<any>` method can be used to send a custom commands to the server.
+In the event a redis command you wish to use is not implemented yet, the `sendCommand(command: string, args: any[]) => Promise<any>` method can be used to send a custom commands to the server.
 ## See also
 
 - [Benchmarking Redis with k6](https://k6.io/blog/benchmarking-redis-with-k6/)
```

Comment:
Not certain we should have this link as this blog post is ancient and not using the curretn API. Maybe  we should update it as well with a notice that this API is no longer the one in the article

Reply:
Indeed 👍🏻